AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -175,6 +181,8 @@ - (void)updateTitle
     _titleNode.attributedString = newTitle;
     [self setNeedsLayout];
   }
+
+  self.accessibilityLabel = _titleNode.accessibilityLabel;
```

Comment:
Not all buttons have a title, and not all titles may have an accessibilityLabel.  This method might get called and overwrite the label in the case of an image-only button where the programmer directly set the accessibilityLabel for the button